src/main/java/loanbook/model/loan/Email.java

```diff
@@ -38,6 +38,10 @@ public Email(String email) {
      * Check if user's new {@code Email} is a valid Gmail.
      */
     public static boolean isValidGmail(String email) {
+        if (email.length() < 16) {
```

Comment:
Why is 16 an invalid gmail?

Reply:
Gmail's local part should have at least 6 chars, 6 + length of "@gmail.com" is 16, so the length of a valid gmail should not be less than 16.

Reply:
Why is this check necessary?

Reply:
If the user set a user email like "123@gmail.com", the app will accept it, but this gmail does not follow the format of Gmail.

Reply:
Should we write a comment to explain? Also, Can this be done in the regex below?

Reply:
Fixed.